src/main/java/net/mcreator/ui/init/EntityAnimationsLoader.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2021, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.init;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import net.mcreator.io.FileIO;
+import net.mcreator.plugin.PluginLoader;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gradle.internal.FileUtils;
+
+import java.util.*;
+import java.util.regex.Pattern;
+
+public class EntityAnimationsLoader {
+
+	private static final Logger LOG = LogManager.getLogger("Entity Animations loader");
+
+	private static final LinkedHashMap<String, JsonArray> entityAnimations = new LinkedHashMap<>();

```

Comment:
At loadEntityAnimations step, convert JsonArray to string array or list so JSON parsing is not needed later.